release_notes/45-release-notes.adoc

```diff
@@ -404,4 +405,27 @@ This release includes notable technical changes and bug fixes.
 * After upgrading to {product-title-short} 4.5.0, for nodes with more than 64 cores, Collector displayed an error about the ring buffer size not being allowed. This problem has been fixed.
 * The network graph was broken for users with clusters running Google Kubernetes Engine (GKE) version 1.29 and later due to a change from Google to use the managed public IP address range `34.118.224.0/20` for default networking services. The network graph marked this IP address range as external. This problem has been fixed.
 
-include::modules/image-versions.adoc[leveloffset=+1]
+[id="about-release-452_{context}"]
+== About release 4.5.2
+
+*Release date*: 09 September 2024
+
+This release of {product-title-short} introduces the following change:
+
+* You can modify the CVE management process to temporarily move orphaned CVEs to a tombstone table to prevent immediate removal and preserve accurate discovery times.
```

Comment:
I think more information is needed for this one. I don't understand what a tombstone table is, or how you move CVEs to them. Is this through the GUI? How do you modify the CVE management process?

Reply:
I documented based on the following ticket:
https://issues.redhat.com/browse/ROX-25519
@dvail Could you please help us with a better text? Thx!

Reply:
+1 to @dvail's suggestion.  We do not need to communicate implementation details here. But we can add that this feature is guarded by an env var `ROX_ORPHANED_CVES_KEEP_ALIVE` and the user will need to set it to true to prevent immediate removal of orphaned node CVEs.